src/components/client-list/client-list.tsx

```diff
@@ -120,21 +121,30 @@ const ClientListLinkButton = styled(LinkButton)`
 
 interface ClientListEntryProps {
   name: string;
+  nameEN: string;
   start: string;
   path: string;
 }
 
 const ClientListEntry: React.FC<ClientListEntryProps> = ({
   name,
+  nameEN,
   start,
   path,
 }) => {
+  const { t } = useTranslation();
+  const { language } = useI18next();
+  const localizedName = 'en' === language ? nameEN : name;
   return (
     <ClientListEntryLi>
       <ClientListEntryLink to={path}>
-        <StyledTitle title={`Show client: ${name}`}>{name}</StyledTitle>
-        <StyledTimestamp>Seit {formatDate(start, 'MMMM y')}</StyledTimestamp>
-        <StyledArrow title={`Show client: ${name}`}>
+        <StyledTitle title={`Show client: ${localizedName}`}>
```

Comment:
I think we need to put the whole string into a translation with a variable. 

This should look like
```js
t('some.key', { clientName: localizedName })
```